freeipa/src/main/java/com/sequenceiq/freeipa/controller/FreeIpaV1Controller.java

```diff
@@ -93,6 +98,16 @@ public DescribeFreeIpaResponse describe(String environmentCrn) {
         return freeIpaListService.list(accountId);
     }
 
+    @Override
+    public HealthDetailsFreeIpaResponse healthDetails(String environmentCrn) {
+        try {
+            String accountId = crnService.getCurrentAccountId();
+            return freeIpaHealthDetailsService.getHealthDetails(environmentCrn, accountId);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
```

Comment:
this doesn't seem right. please use some custom Exception and if necessary create a converter for it also. thanks

Reply:
changed